specs/signature-specification.md

```diff
@@ -323,44 +281,43 @@ Signatures associated with these artifacts require broad portability.
 E.g. Images for containerized applications and services used within an organization, or shared with limited authorized parties.
 Based on user requirements a private artifact can have different levels of portability, the signature’s portability should at least match the the artifact’s portability.
 
-*Notary v2 signature portability* is based on the following
+*Notary signature portability* is based on the following
 
-**Signature discovery**
+#### Signature discovery
 
-Notary v2 addressed signature discovery by storing signatures in the same registry (location) where an artifact is present.
+Notary addressed signature discovery by storing signatures in the same registry (location) where an artifact is present.
 This is supported through [OCI Distribution Referrers API][oci-distribution-referrers] which allows reference artifacts such as signatures, SBOMs to be associated with existing artifacts like Images.
-Notary v2 allows multiple signatures to be associated with an artifact, and clients may automatically push signatures for an artifact to a destination registry when a signed artifact moves from one registry to other.
+Notary allows multiple signatures to be associated with an artifact, and clients may automatically push signatures for an artifact to a destination registry when a signed artifact moves from one registry to other.
 
-**Verification requirements**
+#### Verification requirements
 
-Notary v2 supports a range of signed artifacts intended for public and private distribution.
-Signatures generated by Notary v2 without extended signature attributes marked critical can be verified in any environment where Notation client or another Notary v2 standards compliant verification tool is available, without any additional dependencies.
+Notary supports a range of signed artifacts intended for public and private distribution.
+Signatures generated by Notary without extended signature attributes marked critical can be verified in any environment where Notation client or another Notary standards compliant verification tool is available, without any additional dependencies.
```

Comment:
I would change to _Signatures generated without extended..._ and _where Notation client or another Notary Project standards compliant verification tool is available..._